app/routes.py

```diff
@@ -1,2 +1,48 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, flag):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.flag = flag
+
+planets = [ 
+Planet(0, "jupiter", "largest planet in solar system", False),
+Planet(1, "mercury", "smallest planet in solar system", False), 
+Planet(2, "mars", "most explored planet in solar system", True)
+]
+
+planet_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planet_bp.route("", methods=["GET"])
+def list_planets():
+    planet_list = []
+    for planet in planets:
+        planet_list.append(dict(
```

Comment:
Notice that both here and in the single planet endpoint, we essentially need to build the same dictionary result. In the live code, we introduced an instance method in `Cat` to handle doing this. Learn explicitly covers this in part 7, but if you'd like your code to agree with the live code a little more closely, feel free add that helper now.